src/hooks/useIsMobile.ts

```diff
@@ -0,0 +1,28 @@
+import { detect } from 'detect-browser'
+import { ref, onMounted, onUnmounted } from 'vue'
+
+export function useIsMobile() {
+  const isMobile = ref(false)
+
+  function isMobileCheck() {
+    const browser = detect()
+    const isMobileDevice =
+      browser && browser.os ? ['android', 'ios'].includes(browser.os.toLowerCase()) : false
```

Comment:
Just a suggestion: add "blackberry" and "Windows Mobile". Some people still use it for security reasons.

Reply:
Added